src/Api/Services/Notifications/NotificationsResponseManager.php

```diff
@@ -3,7 +3,67 @@
 namespace App\Api\Services\Notifications;
 
 use App\Api\Services\Base\AbstractResponseManager;
+use App\Api\Services\ResponseCache\ResponseCacheManager;
+use App\DB\Entity\User\Notifications\CatroNotification;
+use App\DB\Entity\User\Notifications\CommentNotification;
+use App\DB\Entity\User\Notifications\FollowNotification;
+use App\DB\Entity\User\Notifications\LikeNotification;
+use App\DB\Entity\User\Notifications\NewProgramNotification;
+use App\DB\Entity\User\Notifications\RemixNotification;
+use App\DB\Entity\User\User;
+use App\DB\EntityRepository\User\Notification\NotificationRepository;
+use OpenAPI\Server\Model\NotificationsCountResponse;
+use OpenAPI\Server\Service\SerializerInterface;
+use Symfony\Contracts\Translation\TranslatorInterface;
 
 final class NotificationsResponseManager extends AbstractResponseManager
 {
+  private NotificationRepository $notification_repository;
+
+  public function __construct(TranslatorInterface $translator,
+                                SerializerInterface $serializer,
+                                ResponseCacheManager $response_cache_manager,
+                                NotificationRepository $notification_repository)
+  {
+    parent::__construct($translator, $serializer, $response_cache_manager);
+    $this->notification_repository = $notification_repository;
+  }
+
+  public function createNotificationsCountResponse(User $user): NotificationsCountResponse
+  {
+    $user->getFollowNotificationMentions();
+
+    $notifications_all = $this->notification_repository->findBy(['user' => $user]);
```

Comment:
With a lot of notifications, this can easily become a huge performance overload. Every notification is fetched 1 by 1.
This could also be done by a single query.